Assets/script.js

```diff
@@ -77,6 +82,35 @@ $(function() {
     $("#calendar-box").datepicker();
   } );
 
+//Modal page
+//Page 1 - Capture username input in variables
+$('#save-name').on("click", function(event) {
+    event.preventDefault();
+    userName = $('#username').val();
+    console.log("Username: " + userName);
+});
+
+//Page 2 - Capture sleep goal inputs (total in minutes) in variables 
+$('#save-sleep').on("click", function(event) {
+    event.preventDefault();
+    sleepGoal = parseInt(($('#sleep-goal-hour').val()*60), 10) + parseInt($('#sleep-goal-minute').val(), 10);
+    console.log("Sleep Goal: " + sleepGoal + " minutes");
```

Comment:
Hi Eitel, why do we have the value 10? is this a set sleepgoal =10hrs?

Reply:
Aw no, the 10 is referring to the decimal base (0-9).